README.md

```diff
@@ -42,8 +43,8 @@ sub init()
   ' Create the AblyTask
   m.ablyTask = createObject("roSGNode", "AblyTask")
 
-  ' Assignee the channel you wish to subscribe to
-  m.ablyTask.channel = "[product:ably-bitflyer/bitcoin]bitcoin:jpy"
+  ' Assignee the channels you wish to subscribe to
+  m.ablyTask.channels = ["[product:ably-bitflyer/bitcoin]bitcoin:jpy"]
```

Comment:
To make it clearer this is an array, perhaps include more than one channel in this example?

Reply:
https://github.com/ably/ably-roku/pull/5/commits/88e4d23ef2f46a9af8d3add85803bbda3e9eb71b